dataplane/Makefile

```diff
@@ -26,3 +26,25 @@ build.image:
 load.image: build.image
 	kind load docker-image $(IMAGE):$(TAG) --name $(KIND_CLUSTER) && \
 		kubectl -n blixt-system rollout restart daemonset blixt-dataplane
+
+.PHONY: build.bytecode.images
+build.bytecode.images: build
+	docker build \
+      --build-arg PROGRAM_NAME=blixt-tc-ingress \
+      --build-arg BPF_FUNCTION_NAME=tc_ingress \
+      --build-arg PROGRAM_TYPE=tc \
+      --build-arg BYTECODE_FILENAME=loader \
+      -f https://raw.githubusercontent.com/bpfd-dev/bpfd/main/packaging/container-deployment/Containerfile.bytecode \
+	  ./target/bpfel-unknown-none/debug -t quay.io/bpfd-bytecode/blixt-tc-ingress:latest
+	docker build \
+      --build-arg PROGRAM_NAME=blixt-tc-egress \
+      --build-arg BPF_FUNCTION_NAME=tc_egress \
+      --build-arg PROGRAM_TYPE=tc \
+      --build-arg BYTECODE_FILENAME=loader \
+      -f https://raw.githubusercontent.com/bpfd-dev/bpfd/main/packaging/container-deployment/Containerfile.bytecode \
+	  ./target/bpfel-unknown-none/debug -t quay.io/bpfd-bytecode/blixt-tc-egress:latest
+
+.PHONY: push.bytecode.images
+push.bytecode.images: build.bytecode.images
+	docker push quay.io/bpfd-bytecode/blixt-tc-egress:latest
+	docker push quay.io/bpfd-bytecode/blixt-tc-ingress:latest
```

Comment:
Similar to above, I guess we just may want to consolidate our images under one room, or at the very least get this set up so that CI can actually push these because I assume right now it has to be done manually?

At minimum I think we should just spin off an issue and link it here pointing out that we need to do some maintenance regarding where images are pushed, could perhaps just add the context to #106